src/tunacode/core/agents/resume/summary.py

```
@@ -94,20 +92,48 @@ def is_summary_message(message: Any) -> bool:
+    """
+    from pydantic_ai.messages import ModelRequest, SystemPromptPart
+
+    summary_content = summary.to_marker_text()
+    system_part = SystemPromptPart(content=summary_content)
+    return ModelRequest(parts=[system_part])
+
+
+def should_compact(
+    messages: list[Any],
+    model_name: str,
+    local_mode: bool = False,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)